java/clients/src/test/java/sleeper/clients/admin/InstanceConfigurationScreenTest.java

```diff
@@ -277,7 +275,6 @@ void shouldOrderKnownPropertiesInTheOrderTheyAreDefinedInTheirGroups() throws Ex
             // Then
             assertThat(output).containsSubsequence(
                     "sleeper.optional.stacks",
-                    "sleeper.default.fs.s3a.readahead.range",
                     "sleeper.default.page.size");
```

Comment:
The test isn't valid after this change. Its purpose is to show that the property groups are displayed in the order they're defined, and properties are displayed in the order they're defined within the group. It's not testing the second half of that any more.